Src/Plugins/Tests.SalAnalyzer/SEC105.CorrectUseOfSalTests.cs

```diff
@@ -0,0 +1,24 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Microsoft.CodeAnalysis.Sarif.PatternMatcher.Plugins.SalAnalyzer
+{
+    public class CorrectUseOfSalTests : EndToEndTestsSalAnalyzer
+    {
+        public CorrectUseOfSalTests(ITestOutputHelper outputHelper) : base(outputHelper)
+        {
+        }
+
+        protected override string RuleId => "SEC105";
+
+        // TODO the test project is targeting netcoreapp3.1
```

Comment:
>// TODO the test project is targeting netcoreapp3.1 [](start = 8, length = 51)

you can remove this comment. we are  testing the library. So, it is netstandard2.0 or net4772

Reply:
Removed.

---
In reply to: [604483058](https://github.com/microsoft/sarif-pattern-matcher/pull/347#discussion_r604483058) [](ancestors = 604483058)